File: databases/loja.py
from utills import *
conn,cur=start("loja")
produtos_p="produto_id integer primary key autoincrement,nome text,valor real,categoria_id integer,constraint fk_categorias foreign key (categoria_id) references categorias(categoria_id)"
criarTabela("categorias","categoria_id integer primary key, categoria text")
criarTabela("produtos",produtos_p)
categorias=[(1,"mercearia"),
            (2,"frutas"),
            (3,"açougue"),
            (4,"higiene"),
            (5,"limpeza")]
produtos=[("feijão",5.10,1),
        ("cuscuz",0.93,1),
        ("maça",7.48,2),
        ("carne de sol",22.98,3),
        ("sabonete xerin",1.10,4),
        ("creme dental dente azul 90g",8.78,4),
        ("sabão em pó xô 'nhaca pôdji 400g",9.89,5)]
insertInto("categorias","categoria_id,categoria",categorias)
insertInto("produtos","nome,valor,categoria_id",produtos)
#cur.executemany("insert into produtos (nome,valor,categoria_id) values (?,?,?)",produtos)
buscaTabela("categorias")
buscaTabela("produtos")
conn.commit()
conn.close()

File: databases/Banco/src/tabela/loja.java
package tabela;

import java.io.FileNotFoundException;
public class loja 
{

	public static void main(String[] args) throws FileNotFoundException 
	{
		//Faz conexão
		Conexao con= new Conexao();
		String s1="categorias";
		String p1="produtos";
		con.drop(p1);
		con.drop(s1);
		con.createTable(s1,"categoria_id integer primary key, categoria text");
		String p="produto_id serial,nome text,valor real,categoria_id integer,constraint fk_categorias foreign key (categoria_id) references categorias(categoria_id)";
		con.createTable(p1,p);
		//Adiciona em tabela

		con.fileInto("src/tabela/categorias",s1,"categoria_id,categoria");
		con.fileInto("src/tabela/produtos",p1,"nome,valor,categoria_id");
		System.out.println("Buscando categorias...Concluido!");
		con.mostrarBusca(con.busca("select * from categorias"));
		
		System.out.println("Buscando produtos...Concluido!");
		con.mostrarBusca(con.busca("select * from produtos"));		
	}

}
